### ISSUE TITLE
pushState on firefox not working?

### ISSUE THREAD

**Issue by mmahalwy** [yahoo/fluxible-router#49]
I am having problems with my app on production with navigation when using firefox. Take a look at alpha.quran.com. If you like on any of the links, all the react world stuff works, fetches data, etc. but does not change the url and therefore everything dependent on it. I am still trying to dig further into if it's a fluxible-router problem on firefox, a reactjs problem or where the source is exactly. I don't have the same issue on safari and chrome.

Thanks!

**Comment by mridgway** [comment]
Seems to be working: Firefox version 38.0.5

**Comment by mmahalwy** [comment]
Hmm, i think firstly, that's awesome that it's working for you, it should work (fingers crossed) but let me try again and get the version number

**Comment by mmahalwy** [comment]
Working
